components/job-orchestration/job_orchestration/executor/query/utils.py

```
@@ -98,6 +99,12 @@ def sigterm_handler(_signo, _stack_frame):
         logger.info(f"{task_name} task {task_id} completed for job {job_id}")
 
     clo_log_file.close()
+    # Dump the stderr log so it's visible in container logs
+    if clo_log_path.stat().st_size > 0:
+        logger.error(
+            f"Contents of {clo_log_path.name}:\n"
+            f"{clo_log_path.read_text()}"
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)